Xamarin.Essentials/HapticFeedback/HapticFeedback.ios.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Threading.Tasks;
+using UIKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class HapticFeedback
+    {
+        internal static bool IsSupported => true;
+
+        static async Task PlatformExecute(HapticFeedbackType type)
+        {
+            switch (type)
+            {
+                case HapticFeedbackType.LongPress:
+                    PlatformLongPress();
+                    break;
+                default:
+                    PlatformClick();
+                    break;
+            }
+            await Task.FromResult(0);
```

Comment:
Maybe use `Task.CompletedTask`. And no need to async. Just return it directly.

Reply:
I added everywhere "await Task.FromResult(0)" only because "await"is used on UWP. Tell me how best to remove it.